<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BisectWindow</class>
 <widget class="QDialog" name="BisectWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>692</width>
    <height>319</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Bisector Window</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Splitting Strategy:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QRadioButton" name="rbIDRange">
          <property name="text">
           <string>ID Range</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbIDLower"/>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>TO</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="cbIDUpper"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="1,10,3">
        <item>
         <widget class="QRadioButton" name="rbFrameNumber">
          <property name="text">
           <string>Frame Number</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="slideFrameNumber">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editFrameNumber"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,10,3">
        <item>
         <widget class="QRadioButton" name="rbPercentage">
          <property name="text">
           <string>Percentage</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="slidePercentage">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10000</number>
          </property>
          <property name="singleStep">
           <number>10</number>
          </property>
          <property name="pageStep">
           <number>100</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editPercentage"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QRadioButton" name="rbBusNum">
          <property name="text">
           <string>Bus Number</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="editBusNum"/>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="title">
      <string>Save Which Side:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_5">
      <item>
       <widget class="QRadioButton" name="rbLowerSection">
        <property name="text">
         <string>Lower Section</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="rbUpperSection">
        <property name="text">
         <string>Upper Section</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_6">
     <item>
      <widget class="QLabel" name="label_4">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Number of Frames in Main List:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelMainListNum">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_5">
       <property name="font">
        <font>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Saved Frames after split:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelSplitNum">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>0</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_7">
     <item>
      <widget class="QPushButton" name="btnCalculate">
       <property name="text">
        <string>Calculate Split</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSaveFrames">
       <property name="text">
        <string>Save split frames to a new file</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnReplaceFrames">
       <property name="text">
        <string>Replace main list with split frames</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>rbIDRange</tabstop>
  <tabstop>cbIDLower</tabstop>
  <tabstop>cbIDUpper</tabstop>
  <tabstop>rbFrameNumber</tabstop>
  <tabstop>slideFrameNumber</tabstop>
  <tabstop>editFrameNumber</tabstop>
  <tabstop>rbPercentage</tabstop>
  <tabstop>slidePercentage</tabstop>
  <tabstop>editPercentage</tabstop>
  <tabstop>rbLowerSection</tabstop>
  <tabstop>rbUpperSection</tabstop>
  <tabstop>btnCalculate</tabstop>
  <tabstop>btnSaveFrames</tabstop>
  <tabstop>btnReplaceFrames</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
